### ISSUE TITLE
Not available for Android

### ISSUE THREAD

**Issue by Jazzinghen** [da2x/amp2html#22]
I tried to install this extension of Firefox for Android and the Firefox extensions website told me it was not available for that platform.

**Comment by da2x** [comment]
Firefox released a new version that only supports a handful of extensions. It’s now up to Mozilla to whitelist extensions that work on mobile. I’m sorry, but I can’t do anything about this situation.

You can still use whatever extensions you want by [switching to the less stable Firefox Nightly](https://www.androidpolice.com/2020/09/30/firefox-for-android-brings-back-general-extensions-support-heres-how-to-set-it-up/).

**Comment by Jazzinghen** [comment]
Oh, I see. I didn't know that, thanks for telling me.
Do you think that I should report this to them or it would be useless?

**Comment by Regenhardt** [comment]
Do you have a way to request your add-on be whitelisted? At first I thought ok they're working on it, but enough time passed now and nothing happened.

**Comment by da2x** [comment]
No, there is no process like that. The extension review team at Mozilla arbitrarily select some extensions they like, and those are the only ones anyone gets to install.

**Comment by leonixyz** [comment]
I am using your extension on Android for rewriting Twitter links and it works perfectly. Thanks for your effort
